doc/Language/variables.pod6

```diff
@@ -328,7 +327,7 @@ to that block. This is because the placeholder variables are sorted in Unicode
 order. If you have self-declared a parameter using C<$^a> once, you may refer
 to it using only C<$a> thereafter.
 
-Although it is possible to use nearly any valid identifier as a placeholder
+Although it's possible to use nearly any valid identifier as a placeholder
```

Comment:
I'm not getting this, unless you're simply saying, find a way not to use it is (or it's) at the beginning of the sentence.